rack servers instance rebuild broken

```
11:46:51 (master=) kyle6475@puter ~/go/src/github.com/jrperritt/rack$ go build -o $GOPATH/bin/rack && rack servers instance rebuild
rebuild flag redefined: name
panic: rebuild flag redefined: name

goroutine 1 [running]:
flag.(*FlagSet).Var(0xc20805c5a0, 0x67e010, 0xc20802d8b0, 0x3d5b50, 0x4, 0x45df50, 0x4d)
	/usr/local/Cellar/go/1.4.2/libexec/src/flag/flag.go:679 +0x494
flag.(*FlagSet).StringVar(0xc20805c5a0, 0xc20802d8b0, 0x3d5b50, 0x4, 0x0, 0x0, 0x45df50, 0x4d)
	/usr/local/Cellar/go/1.4.2/libexec/src/flag/flag.go:582 +0xcf
flag.(*FlagSet).String(0xc20805c5a0, 0x3d5b50, 0x4, 0x0, 0x0, 0x45df50, 0x4d, 0x4)
	/usr/local/Cellar/go/1.4.2/libexec/src/flag/flag.go:595 +0x86
github.com/codegangsta/cli.func·013(0x3d5b50, 0x4)
	/Users/kyle6475/go/src/github.com/codegangsta/cli/flag.go:348 +0x62
github.com/codegangsta/cli.eachName(0x3d5b50, 0x4, 0xc208092838)
	/Users/kyle6475/go/src/github.com/codegangsta/cli/flag.go:54 +0x11e
github.com/codegangsta/cli.StringFlag.Apply(0x3d5b50, 0x4, 0x0, 0x0, 0x45df50, 0x4d, 0x0, 0x0, 0xc20805c5a0)
	/Users/kyle6475/go/src/github.com/codegangsta/cli/flag.go:349 +0x18f
github.com/codegangsta/cli.(*StringFlag).Apply(0xc20800a6c0, 0xc20805c5a0)
	<autogenerated>:33 +0xaa
github.com/codegangsta/cli.flagSet(0x3d76b0, 0x7, 0xc20807a280, 0xc, 0x14, 0xc20803fa70)
	/Users/kyle6475/go/src/github.com/codegangsta/cli/flag.go:45 +0x133
github.com/codegangsta/cli.Command.Run(0x3d76b0, 0x7, 0x0, 0x0, 0x0, 0x0, 0x0, 0xc208064120, 0x86, 0x4182d0, ...)
	/Users/kyle6475/go/src/github.com/codegangsta/cli/command.go:60 +0x3d2
github.com/codegangsta/cli.(*App).RunAsSubcommand(0xc2080821e0, 0xc2080880e0, 0x0, 0x0)
	/Users/kyle6475/go/src/github.com/codegangsta/cli/app.go:255 +0xc2a
github.com/codegangsta/cli.Command.startApp(0x3ebdb0, 0x8, 0x0, 0x0, 0x0, 0x0, 0x0, 0x42d390, 0x23, 0x0, ...)
	/Users/kyle6475/go/src/github.com/codegangsta/cli/command.go:183 +0x505
github.com/codegangsta/cli.Command.Run(0x3ebdb0, 0x8, 0x0, 0x0, 0x0, 0x0, 0x0, 0x42d390, 0x23, 0x0, ...)
	/Users/kyle6475/go/src/github.com/codegangsta/cli/command.go:45 +0x14ee
github.com/codegangsta/cli.(*App).RunAsSubcommand(0xc2080820f0, 0xc208088000, 0x0, 0x0)
	/Users/kyle6475/go/src/github.com/codegangsta/cli/app.go:255 +0xc2a
github.com/codegangsta/cli.Command.startApp(0x3d80b0, 0x7, 0x0, 0x0, 0x0, 0x0, 0x0, 0x4186d0, 0x1c, 0x0, ...)
	/Users/kyle6475/go/src/github.com/codegangsta/cli/command.go:183 +0x505
github.com/codegangsta/cli.Command.Run(0x3d80b0, 0x7, 0x0, 0x0, 0x0, 0x0, 0x0, 0x4186d0, 0x1c, 0x0, ...)
	/Users/kyle6475/go/src/github.com/codegangsta/cli/command.go:45 +0x14ee
github.com/codegangsta/cli.(*App).Run(0xc208082000, 0xc20800a000, 0x4, 0x4, 0x0, 0x0)
	/Users/kyle6475/go/src/github.com/codegangsta/cli/app.go:158 +0xd0c
main.main()
	/Users/kyle6475/go/src/github.com/jrperritt/rack/main.go:23 +0x16c

goroutine 2 [runnable]:
runtime.forcegchelper()
	/usr/local/Cellar/go/1.4.2/libexec/src/runtime/proc.go:90
runtime.goexit()
	/usr/local/Cellar/go/1.4.2/libexec/src/runtime/asm_amd64.s:2232 +0x1

goroutine 3 [runnable]:
runtime.bgsweep()
	/usr/local/Cellar/go/1.4.2/libexec/src/runtime/mgc0.go:82
runtime.goexit()
	/usr/local/Cellar/go/1.4.2/libexec/src/runtime/asm_amd64.s:2232 +0x1

goroutine 4 [runnable]:
runtime.runfinq()
	/usr/local/Cellar/go/1.4.2/libexec/src/runtime/malloc.go:712
runtime.goexit()
	/usr/local/Cellar/go/1.4.2/libexec/src/runtime/asm_amd64.s:2232 +0x1
```

--- a/serverscommands/instancecommands/rebuild.go
+++ b/serverscommands/instancecommands/rebuild.go
@@ -34,10 +34,6 @@ func flagsRebuild() []cli.Flag {
 			Usage: "[required] The server's administrative password.",
 		},
 		cli.StringFlag{
-			Name:  "name",
-			Usage: "[optional] The name for the rebuilt server.",
-		},
-		cli.StringFlag{
 			Name:  "accessIPv4",
 			Usage: "[optional] The IPv4 address for the rebuilt server.",
 		},

